<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parametre</class>
 <widget class="QDialog" name="parametre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>parametre</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/resource/image/Fichier_4.png</normaloff>:/image/resource/image/Fichier_4.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>notification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="toolTip">
         <string>permet de definire lactivation des notification visuelle</string>
        </property>
        <property name="whatsThis">
         <string>permet de definire lactivation des notification visuelle</string>
        </property>
        <property name="accessibleDescription">
         <string>permet de definire lactivation des notification visuelle</string>
        </property>
        <property name="text">
         <string>notification visuelle</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="toolTip">
         <string>permet de definire lactivation des notification sonore</string>
        </property>
        <property name="accessibleDescription">
         <string>permet de definire lactivation des notification sonore</string>
        </property>
        <property name="text">
         <string>notification sonore</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>son par default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blup blup</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>selectioner une musique</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>plus d'informantion</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/image/resource/image/analyse.png</normaloff>:/image/resource/image/analyse.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>contact</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="site">
        <property name="text">
         <string>le site</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/Anantasite.png</normaloff>:/image/resource/image/Anantasite.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="discord">
        <property name="text">
         <string>le discord (aide/bug/probleme de traduction)</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/discord logo.png</normaloff>:/image/resource/image/discord logo.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="tweter">
        <property name="text">
         <string>le twiter</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/Twitter_bird_logo_2012.svg.png</normaloff>:/image/resource/image/Twitter_bird_logo_2012.svg.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="don">
        <property name="text">
         <string>faire un don</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/image0.jpg</normaloff>:/image/resource/image/image0.jpg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>sucées</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>voir mes succés</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/success.png</normaloff>:/image/resource/image/success.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>nombre de message :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>serveur</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>sauvgardée les message</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>suprimée tout les message</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/supprimer.png</normaloff>:/image/resource/image/supprimer.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>interface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QComboBox" name="setcollor">
        <item>
         <property name="text">
          <string>blanc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blanc ultra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>noir</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>oled</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkboxmodecondensee">
        <property name="text">
         <string>activer la transparence du mode condensée</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="language">
        <item>
         <property name="text">
          <string>francais</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>anglais</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/Blue_question_mark_icon.svg.png</normaloff>:/image/resource/image/Blue_question_mark_icon.svg.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>langue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/image/resource/image/Blue_question_mark_icon.svg.png</normaloff>:/image/resource/image/Blue_question_mark_icon.svg.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>theme</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
